manual/core/integration/README.md

```diff
@@ -496,6 +497,22 @@ don't use geospatial types anywhere in your application, you can exclude the dep
 </dependency>
 ```
 
+Starting with driver 4.14.0 Esri has been changed to an optional dependency.  You no longer have to
+explicitly exclude the dependency if it's not used, but if you do wish to make use of the Esri
+library you must now explicitly specify it as a dependency :
+
+```xml
+<dependency>
+  <groupId>com.esri.geometry</groupId>
+  <artifactId>esri-geometry-api</artifactId>
+  <version>${esri.version}</version>
+</dependency>
+```
+
+The dependency specification above will introduce a dependency on version 1.2.1 of the Esri
```

Comment:
Small suggestion to better link this to the above snippet:

> In the dependency specification above, `esri.version` should be any version in the range `[1.2,2.0)`, for example 1.2.1. While this version is now old, it is the only version that is guaranteed to be fully compatible with DSE.

Reply:
Oh, I like that _much_ better.  I tried to re-phrase what I had a couple of times but I never really liked what I came up with... I'll switch to your language instead.

Reply:
I basically used the language you provided with a (hopefully small and non-obtrusive) tweak to avoid specifying the version range with the [) syntax; I was worried it might be confusing to some users.  Take a look and let me know if you're okay with that usage.

Reply:
That's excellent, thanks!